services/osparc-gateway-server/tests/unit/test_utils.py

```diff
@@ -162,6 +179,7 @@ async def test_create_service_config(
     network_id = faker.uuid4()
     cmd = faker.pystr()
     fake_labels = faker.pydict()
+    fake_placement = {"Constraints": [f"node.hostname=={faker.pystr()}"]}
```

Comment:
FYI: ``faker.hostname`` is available: https://faker.readthedocs.io/en/master/providers/faker.providers.internet.html#faker.providers.internet.Provider.hostname

Reply:
done